Source: Konstantinn Bonnet <qu7uux@gmail.com>
Upstream: http://sourceforge.net/p/jfs/bugs/286/
Reason: Fix missing #include causing a compilation error
--
diff -Naur a/libfs/devices.h b/libfs/devices.h
--- a/libfs/devices.h	2014-01-06 00:08:59.421776990 +0100
+++ b/libfs/devices.h	2014-01-06 00:09:07.399776635 +0100
@@ -18,6 +18,8 @@
 #ifndef H_DEVICES
 #define H_DEVICES
 
+#include <stdint.h>
+
 #define GET	0
 #define PUT	1
 #define VRFY	2
